css/components/Card.css

```diff
@@ -0,0 +1,161 @@
+@import "../style.css";
+
+.container{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.badge-card, .dismiss-card, .text-overlay-card, .text-card, .horizontal-card, .shadow-card{
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 200px;
+    height: 250px;
+    border: 1px solid var(--badge-offline);
+    background-color: var(--badge-offline);
+    border-radius: 1rem;
+    margin: .5rem;
```

Comment:
Rather than combining all the classes and giving a common style, you can have a class called card give your common style to that class and use that card class where needed.

Reply:
conflict resolved